src/account/components/AccountLogin.js

```diff
@@ -10,10 +11,15 @@ import PageLoader from 'common/components/PageLoader';
 
 import logo from 'assets/logo.svg';
 
+const appendReferrer = (url, referrer) =>
+  referrer ? `${url}&state=${referrer}` : url;
+
 const AccountForm = () => {
   const dispatch = useDispatch();
   const { t } = useTranslation();
+  const [searchParams] = useSearchParams();
   const { fetching, urls } = useSelector(state => state.account.login);
+  const referrer = searchParams.get('referrer');
```

Comment:
question: should this query parameter match L15 above?

Reply:
@paulschreiber This is the query parameter exposed in the browser, the one in L15 is the one sent to the API

Reply:
I have no preference using `referrer` or something else here. So, `referrer` is ok.